tests/e2e/scale/test_scale_amq.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import E2ETest, performance
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.amq import AMQ
+from tests.helpers import default_storage_class
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def test_fixture_amq(request):
+
+    amq = AMQ()
+
+    def teardown():
+        amq.cleanup()
+
+    request.addfinalizer(teardown)
+    return amq
+
+
+@performance
+class TestAMQBasics(E2ETest):
+    @pytest.mark.parametrize(
+        argnames=["interface"],
+        argvalues=[
+            pytest.param(
+                constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-1279")
+            )
+        ]
+    )
```

Comment:
Are we going to use this test only for `constants.CEPHBLOCKPOOL` or we have any future scope? 
If no scope then better to remove these lines, add like below in line48
`sc = default_storage_class(interface_type=constants.CEPHBLOCKPOOL)`, also you can add the polarion_id marker something like `@pytest.mark.polarion_id("OCS-2321")` in line 25/26

Reply:
added polarion_id, for now its wil be in scope of constants.CEPHBLOCKPOOL

Reply:
it will be for  ceph for now,  I will change and extend in the future if needed
fixed polarian_id